vault/CHANGELOG.md

```diff
@@ -1 +1,5 @@
 # CHANGELOG - Vault
+
+## 1.0.0 / 2018-06-19
+
+* [Added] add Vault integration. See [#1759](https://github.com/DataDog/integrations-core/pull/1759).
```

Comment:
Super small issue, can we capitalize the "a" in "add" (here as well as the original PR)?

Reply:
Let's make sure we clean the squash description when we squash multiple commits as well